eslint.config.js

```diff
@@ -137,6 +133,33 @@ export default tseslint.config(
       ],
       'import/newline-after-import': 'error',
       'import/no-duplicates': 'error',
+      'no-restricted-imports': [
```

Comment:
This makes it so you can only use these imports inside the `primitives` folder. Let me know what you think of it. Think I kind of like it and maybe it helps avoid the View-soup and helps keep things nicely structured 🤞